packages/docusaurus/src/webpack/client.ts

```diff
@@ -30,6 +31,26 @@ export function createClientConfig(props: Props): Configuration {
       runtimeChunk: true,
     },
     plugins: [
+      // Plugin to force terminate building if errors happened in the client bundle
```

Comment:
Perhaps this tiny plugin needs to be created in a separate file or class (in same file), but not sure about it. But feel free to change that.

Reply:
> fix(v2) -> chore(v2) ?

Not sure exactly how the type of commit here accurately reflects the situation...

Reply:
> Perhaps this tiny plugin needs to be created in a separate file or class (in same file), but not sure about it. But feel free to change that. Something like `ClientBundleCheckerPlugin`??

I feel like you are right here. It can be made a separate plugin. Don't wish to take away your thunder and all your hard work on this but I can try do that if you want? :)